io.github.Nheko-Reborn.Nheko.json

```diff
@@ -0,0 +1,199 @@
+          "disable-shallow-clone": true,
+          "tag": "2.2.2",
+          "type": "git",
+          "url": "https://gitlab.matrix.org/matrix-org/olm.git"
+        },
+        {
+          "dest": ".",
+          "path": "cmake/CMakeLists.txt",
+          "type": "file"
+        },
+        {
+          "dest": "cmake",
+          "path": "cmake/OlmConfig.cmake.in",
+          "type": "file"
+        }
+      ]
+    },
+    {
+      "name": "sodium",
+      "sources": [
+        {
+          "sha256": "eeadc7e1e1bcef09680fb4837d448fbdf57224978f865ac1c16745868fbd0533",
+          "type": "archive",
+          "url": "https://github.com/jedisct1/libsodium/releases/download/1.0.16/libsodium-1.0.16.tar.gz"
+        }
+      ]
+    },
+    {
+      "build-commands": [
+        "./bootstrap.sh --with-libraries=random,thread,system,iostreams,atomic,chrono,date_time,regex --prefix=/app",
+        "./b2 -d0 variant=release link=shared threading=multi --layout=system",
+        "./b2 -d0 install"
+      ],
+      "buildsystem": "simple",
+      "name": "boost",
+      "sources": [
+        {
+          "sha256": "7f6130bc3cf65f56a618888ce9d5ea704fa10b462be126ad053e80e553d6d8b7",
+          "type": "archive",
+          "url": "https://dl.bintray.com/boostorg/release/1.68.0/source/boost_1_68_0.tar.bz2"
+        }
+      ]
+    },
+    {
+      "config-opts": [
+        "-DBUILD_LIB_TESTS=OFF",
+        "-DBUILD_LIB_EXAMPLES=OFF",
+        "-DCMAKE_BUILD_TYPE=Release"
+      ],
+      "buildsystem": "cmake",
+      "name": "mtxclient",
+      "sources": [
+        {
+          "sha256": "e7638d4a8233c0c763d48111fd13e8ad1dcd5f34e3e641b46eaf1bb920b73482",
+          "type": "archive",
+          "url": "https://github.com/nheko-reborn/mtxclient/archive/v0.2.0.tar.gz"
+        },
+        {
+          "dest": "include",
+          "sha256": "ce6b5610a051ec6795fa11c33854abebb086f0fd67c311f5921c3c07f9531b44",
+          "type": "file",
+          "url": "https://github.com/nlohmann/json/releases/download/v3.2.0/json.hpp"
+        }
+      ]
+    },
+    {
+      "config-opts": [
+        "-DCMAKE_BUILD_TYPE=Release",
+        "-DTWEENY_BUILD_DOCUMENTATION=OFF",
+        "-DTWEENY_BUILD_EXAMPLES=OFF"
+      ],
+      "buildsystem": "cmake-ninja",
+      "name": "tweeny",
+      "sources": [
+        {
+          "sha256": "482857256a7235646004682912badb6521d361ed6987c8ebdae7986bf64ce694",
+          "type": "archive",
+          "url": "https://github.com/mobius3/tweeny/archive/43f4130f7e4a67c19d870b60864bc2862c19b81f.tar.gz"
+        }
+      ]
+    },
+    {
+      "config-opts": [
+        "-DCMAKE_BUILD_TYPE=Release",
+        "-DLMDBXX_INCLUDE_DIR=.deps/lmdbxx"
+      ],
+      "buildsystem": "cmake",
+      "name": "nheko",
+      "sources": [
+        {
+          "sha256": "869dea68ad53258cbff4e41bb3e2894742ef90568779787d8eb5391112eb3d48",
+          "type": "archive",
+          "url": "https://github.com/mujx/nheko/archive/v0.6.2.tar.gz"
```

Comment:
this still seems to be using an older release from the other repo?

Reply:
Whoops, I missed this when I updated the file.